lib/Zonemaster/Backend/DB/SQLite.pm

```diff
@@ -96,6 +99,62 @@ sub create_schema {
         'CREATE INDEX IF NOT EXISTS test_results__domain_undelegated ON test_results (domain, undelegated)'
     );
 
+    ####################################################################
+    # LOG LEVEL
+    ####################################################################
+    $dbh->do(
+        "CREATE TABLE IF NOT EXISTS log_level (
+            value INTEGER,
+            level VARCHAR(15),
+
+            UNIQUE (value)
+        )
+        "
+    ) or die Zonemaster::Backend::Error::Internal->new( reason => "SQLite error, could not create 'log_level' table", data => $dbh->errstr() );
+
+    my ( $c ) = $dbh->selectrow_array( "SELECT count(*) FROM log_level" );
+    if ( $c == 0 ) {
+        $dbh->do(
+            "INSERT INTO log_level (value, level)
+            VALUES
+                (-2, 'DEBUG3'),
+                (-1, 'DEBUG2'),
+                ( 0, 'DEBUG'),
+                ( 1, 'INFO'),
+                ( 2, 'NOTICE'),
+                ( 3, 'WARNING'),
+                ( 4, 'ERROR'),
+                ( 5, 'CRITICAL')
+            "
+        );
+    }
+
+    ####################################################################
+    # RESULT ENTRIES
+    ####################################################################
+    $dbh->do(
+        'CREATE TABLE IF NOT EXISTS result_entries (
+            hash_id VARCHAR(16) NOT NULL,
```

Comment:
Why not have the foreign key referring to `test_results.id` instead of `test_results.hash_id`? (see comment above)